include/evmc/evmc.h

```diff
@@ -880,6 +901,9 @@ struct evmc_host_interface
 
     /** Set transient storage callback function. */
     evmc_set_transient_storage_fn set_transient_storage;
+
+    /** Get transaction initcode by hash callback function. */
+    evmc_get_tx_initcode_by_hash_fn get_tx_initcode_by_hash;
```

Comment:
Does it make sense to put it close to `get_tx_context` ?

Reply:
as in, move it up/down in the file? I think so. I can't remember exactly why I positioned stuff where I did anymore.